plugins/catalog-backend/api-report.md

```diff
@@ -306,12 +306,6 @@ export type CatalogEnvironment = {
   permissions: PermissionAuthorizer;
 };
 
-// @public
-export type CatalogPermissionRule = PermissionRule<
```

Comment:
Could be nice to deprecate this, but seeing as there's likely very little usage I think we can make it a direct breaking change. Needs a changeset tho

Reply:
Oh derp, we haven't actually released this yet have we?

Reply:
Nope!